apps/base-docs/docs/pages/chain/differences-between-ethereum-and-base.mdx

```diff
@@ -6,7 +6,7 @@ description: Documentation covering the differences between Ethereum and Base. T
 
 # Differences between Ethereum and Base
 
-Base is built on the [Bedrock](https://stack.optimism.io/docs/releases/bedrock/explainer/) release of the [OP Stack](https://stack.optimism.io/), which is designed from the ground up to be as close to Ethereum as possible. Because of this, there are very few differences when it comes to building on Base and Ethereum.
+Base is built on the [Bedrock](https://docs.optimism.io/stack/rollup/overview) release of the [OP Stack](https://stack.optimism.io/), which is designed from the ground up to be as close to Ethereum as possible. Because of this, there are very few differences when it comes to building on Base and Ethereum.
```

Comment:
@Rozi1vagen Thanks! I'm not sure this is contextually equivalent; can you please update to drop the link? I think the second link to just the OP Stack should be sufficient 👍 

Appreciate you flagging that this other link wasn't working anymore! :)